client/entrypoints/express-checkout/styles.scss

```diff
@@ -0,0 +1,3 @@
+#wc-stripe-express-checkout-element iframe {
+	max-width: unset;
```

Comment:
This fixes an issue on all shortcode powered pages and follows the same approach WooPayments took in https://github.com/Automattic/woocommerce-payments/pull/9094 and discussed with Stripe internallly here: p1720833500937719-slack-C9976E5MJ

**PAY FOR ORDER**
<p align="center">
<img width="1188" alt="Screenshot 2024-09-26 at 4 44 10 pm" src="https://github.com/user-attachments/assets/d6798346-f4ac-405c-a710-cc1a5e622854">
</br>
<sup>Gap on the right hand side of the payment elements.</sup>
</p>

**CHECKOUT**
<p align="center">
<img width="596" alt="Screenshot 2024-09-26 at 5 20 45 pm" src="https://github.com/user-attachments/assets/533088a9-bd45-47c8-bbfe-f8ffee4500f7">
</p> 


**CART**
<p align="center">
<img width="629" alt="Screenshot 2024-09-26 at 5 11 45 pm" src="https://github.com/user-attachments/assets/621d8250-2830-4176-b525-b535ecddc709">
</p>